Makefile

```diff
@@ -202,7 +203,7 @@ eslint: node-babel-eslint node-eslint node-eslint-plugin-html
 
 dev_data:
 	cd $(WPTD_GO_PATH)/util; go get -t ./...
-	go run $(WPTD_GO_PATH)/util/populate_dev_data.go $(FLAGS) -static_dir="$(WPTD_GO_PATH)/webapp/static"
+	go run $(WPTD_GO_PATH)/util/populate_dev_data.go $(FLAGS)
```

Comment:
Why's this change?

Reply:
That flag doesn't exist anymore

Reply:
Oops. It was added in #358 , probably mistakenly. Back in #358 I thought that was the new flag Mark added. Thanks for spotting this.